.pipelines/ci.yml

```diff
@@ -473,6 +473,68 @@ stages:
         python -m pytest --ignore=test_cv2.py --ignore=test_tools_add_pre_post_processing_to_model.py . --verbose
       displayName: Run python test
 
+- stage: WindowsCUDABuilds
+  dependsOn: []
+  jobs:
+  - job: WindowsC
+    pool:
+      name: 'onnxruntime-extensions-Win2022-GPU-A10'
+    variables:
+      ORT_VERSION: '1.16.2'
+    steps:
+      - template: templates/set_winenv.yml
+        parameters:
+          EnvSetupScript: 'set_env_cuda.bat'
+          DownloadCUDA: true
+
+      - script: |
+          nvidia-smi
+          nvcc --version
+          where nvcc
+        displayName: check cuda version
+
+      - task: DownloadGitHubRelease@0
+        inputs:
+          connection: 'GitHub - Release'
+          userRepository: 'microsoft/onnxruntime'
+          defaultVersionType: 'specificTag'
+          version: 'v$(ORT_VERSION)'
+          itemPattern: '*-win-x64-gpu-$(ORT_VERSION)*'
+          downloadPath: '$(Build.SourcesDirectory)'
+        displayName: Download the ONNXRuntime prebuilt package.
+
+      - task: ExtractFiles@1
+        inputs:
+          archiveFilePatterns: '**/*.zip'
+          destinationFolder: '$(Build.SourcesDirectory)'
+          cleanDestinationFolder: false
+          overwriteExistingFiles: true
+        displayName: Unpack ONNXRuntime package.
+
+      - script: |
```

Comment:
Is this step really needed? CMake can locate Visual Studio by itself. CMake doesn't need this. 
If this step is needed, vsdevcmd.bat is not the right bat file to invoke.  It needs to be  something like:
```yaml
- ${{ if eq(parameters.BuildArch, 'x64') }}:
  - script: |
      @echo off
      set vswherepath="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
      for /f "usebackq delims=" %%i in (`%vswherepath% -latest -property installationPath`) do (
        if exist "%%i\VC\Auxiliary\Build\vcvars64.bat" (
          set vcvarsall="%%i\VC\Auxiliary\Build\vcvars64.bat"
        )
      )
      @echo %vcvarsall% will be used as the VC compiler
      @echo ##vso[task.setvariable variable=vcvarsall]%vcvarsall%
    displayName: 'locate vcvarsall via vswhere'
- ${{ if eq(parameters.BuildArch, 'x86') }}:
  - script: |
      @echo off
      set vswherepath="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
      for /f "usebackq delims=" %%i in (`%vswherepath% -latest -property installationPath`) do (
        if exist "%%i\VC\Auxiliary\Build\vcvars32.bat" (
          set vcvarsall="%%i\VC\Auxiliary\Build\vcvars32.bat"
        )
      )
      @echo %vcvarsall% will be used as the VC compiler
      @echo ##vso[task.setvariable variable=vcvarsall]%vcvarsall%
    displayName: 'locate vcvarsall via vswhere'
```

Reply:
No. I've removed this step.